Dockerfile/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV ROS_DISTRO humble
+
+# Make sure everything is up to date before building from source
+RUN apt-get update \
+  && apt-get upgrade -y \
+  && apt-get update && apt-get install -q -y --no-install-recommends \
+    dirmngr \
+    gnupg \
+    lsb-release \
+    wget \
+    curl \
+    git \
+    ca-certificates \
+    build-essential \
+    cmake \
+  && apt-get clean
+
+RUN curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg && \
+  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | tee /etc/apt/sources.list.d/ros2.list > /dev/null && \
+  apt-get update && apt-get install -q -y --no-install-recommends \
+    python3-colcon-ros \
+    python3-colcon-common-extensions \
+    python3-rosdep \
+    python3-vcstool \
+    && apt-get clean
+
+RUN mkdir -p /home/ros2_ws/src \
+    && cd /home/ros2_ws/src \
+    && git clone https://github.com/ARK3r/ros2_control_demos -b humble \
```

Comment:
Should this really be doing a git clone here? This is the project getting cloned so a COPY could work and ensure the latest changes locally are picked up
Should this be cloning a fork?
The branch should be gotten from an arg

Reply:
I think your suggested change makes sense. I'll fix this now.